src/Prometheus.Client.MetricPusher.HostedService.Tests/GlobalUsings.cs

```diff
@@ -0,0 +1,2 @@
+global using Xunit;
```

Comment:
Folder `Prometheus.Client.MetricPusher.HostedService.Tests` exists in `src` and `tests`

![image](https://github.com/prom-client-net/prom-client-metricpusher-hostedservice/assets/1101691/a90a616d-0ccd-40c5-aaa6-39b113a8c28d)

Reply:
@phnx47 
My bad, just deleted it.